examples/api/lib/material/refresh_indicator/refresh_indicator.0.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for RefreshIndicator
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'Flutter Code Sample';
```

Comment:
You should probably customize this to the refresh indicator: it doesn't need to be so generic. Could be "RefreshIndicator Sample", for instance.